.babelrc

```diff
@@ -1,5 +1,16 @@
 {
   "presets": [
     "latest"
-  ]
+  ],
+  "env": {
+    "production": {
+      "presets": [
+        ["latest", {
+          "es2015": {
+            "modules": false
+          }
+        }]
+      ]
+    }
```

Comment:
Note that the production section added above is optional. It's there to disable transpiling modules in the production build since Webpack 2 can parse these to support tree shaking.